frontend/cypress/e2e/0-APLOSE/2 - datasets.cy.js

```diff
@@ -1,48 +1,52 @@
 describe('datasets page', () => {
     beforeEach(() => {
-        cy.login("admin", "osmose29")
-        cy.contains("Datasets").click()
+        cy.session_login('admin', 'osmose29')
+        cy.contains('Datasets').click()
     })
 
-    it('displays 2 datasets by default', () => {
-        cy.get('tr').should('have.length', 3) // Includes table head
+    it('displays 5 datasets by default', () => {
+        cy.get('tr').should('have.length', 6) // Includes table head
     })
 
-    describe('import', () => {
-        beforeEach(() => {
-            cy.contains('Import').click()
-        })
+    // This depends on a proper datawork folder setup
+    // TODO : make it so there is one on CI
+    if(!Cypress.env("CI_ENV")) {
```

Comment:
Why does the env variable name change from the CI configuration "CYPRESS_CI_ENV" ?

Reply:
I actually had named it CI_ENV everywhere but it did not work :sweat_smile:, I saw in the [documentation](https://docs.cypress.io/guides/guides/environment-variables) the following info : 

> Difference between OS-level and Cypress environment variables
> 
> In Cypress, "environment variables" are variables that are accessible via Cypress.env. These are not the same as OS-level environment variables. However, [it is possible to set Cypress environment variables from OS-level environment variables](https://docs.cypress.io/guides/guides/environment-variables#Option-3-CYPRESS_).

Reply:
Good to know, thanks :)